### PR TITLE
provide data in dl samples 

### PR DESCRIPTION
**Pull request by priyankatuteja**
- [x] automate_building_footprint_extraction_using_instance_segmentation
- [x] automate_road_surface_investigation_using_deep_learning
- [x] building_reconstruction_using_mask_rcnn
- [x] detecting_settlements_using_supervised_classification_and_deep_learning
- [x] detecting_swimming_pools_using_satellite_image_and_deep_learning
- [x] extracting_building_footprints_from_drone_data
- [x] extracting_slums_from_satellite_imagery
- [x] feature_categorization_using_satellite_imagery_and_deep_learning
- [x] increase-image-resolution-using-superresolution
- [x] land_cover_classification_using_sparse_training_data
- [x] land_cover_classification_using_unet
- [x] shipwrecks_detection_using_bathymetric_data
- [x] train_a_tensorflow-lite_model_for_identifying_plant_species
- [x] detecting_and_categorizing_brick_kilns_from_satellite_imagery
-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [x] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/building_reconstruction_using_mask_rcnn.ipynb [hunk 1]
```diff
@@ -195,22 +195,25 @@
   },
```
Comment (AtmaMani) [inline samples/04_gis_analysts_data_scientists/building_reconstruction_using_mask_rcnn.ipynb:195]: <p>Should the connection be put back to 'home'?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/789/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

File: samples/04_gis_analysts_data_scientists/detecting_and_categorizing_brick_kilns_from_satellite_imagery.ipynb [hunk 2]
```diff
@@ -153,10 +153,14 @@
   },
```
Comment (AtmaMani) [inline samples/04_gis_analysts_data_scientists/detecting_and_categorizing_brick_kilns_from_satellite_imagery.ipynb:153]: <p>should the connection be 'home'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/789/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Review by AtmaMani — CHANGES_REQUESTED** [review]
A couple of notebooks create GIS connection with url, username, password. Otherwise changes look good.

**Review by rohitgeo — COMMENTED** [review]
Thanks @priyankatuteja for adding the sample data. This should make the samples much more approachable for our users.

**Comment by anneku** [comment]
Thank you for adding all the sample data.

I am unable to access the sample training data for automate_building_footprint_extraction_using_instance_segmentation.ipynb
Automate Building Footprint Extraction using Deep learning

`training_data = gis.content.get('637825446a3641c2b602ee854776ed47')`

"The item you requested cannot be found. The item may have been deleted or you may have entered an incorrect URL."

**Pull request merged**